sycl/include/CL/sycl/detail/spirv.hpp

```diff
@@ -133,21 +133,22 @@ EnableIfNativeBroadcast<T, IdT> GroupBroadcast(T x, IdT local_id) {
   using GroupIdT = typename GroupId<Group>::type;
   GroupIdT GroupLocalId = static_cast<GroupIdT>(local_id);
   using OCLT = detail::ConvertToOpenCLType_t<T>;
+  using WidenedT = conditional_t<
```

Comment:
We should probably have a `FIXME` here saying that widening should be disabled once all backends support all data types

Reply:
Added.